examples/kafka-streams-examples/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+---
+version: '2'
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:3.2.0
```

Comment:
Yes, because when you build a branch the docker images are built with tags: latest and current version in the Makefile. This way we don't have to update the tests for each point release. 

This is only intentional for tests but it works for examples too as the docs always point to the latest release (but I think we need pin the version in the docs as this will break for older versions of docs) .

@samjhecht ^^